test/hooks_test.dart

```diff
@@ -642,97 +643,97 @@ main() {
     });
 
     group('useImperativeHandle -', () {
-      var mountNode = new DivElement();
```

Comment:
Best viewed with whitespace-agnostic diff; besides changing to `forwardRef2`, these were just indented